documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -29,6 +31,14 @@ data, a user works with smaller objects, extracted from the full dataset using
   slice-o-matic), 2-d and 1-d cuts can be easily represented.
 - The entire dataset is often too large to fit in the memory of most computers.
 
+.. note::
+
+    "pixels" in 1D dataset are arranged in such a way, that simple calculation allows 
+    to identify "pixels" contributing into every bin of a ``sqw`` object's ``image``. 
+    ``cut`` uses this information to extract relevant pixels and maintains 
+    the arrangement between bins and "pixels" for resulting ``sqw`` object.
```

Comment:
```suggestion
    The "pixels" can be mapped from their 4-D spatial coordinates into the "image"
     by a transformation determined by the projection. 
     ``cut`` computes the bin to which "pixel" contributes, 
     removing those pixels which do not contribute from the resulting ``sqw``. 
    
    It also sorts the pixels according to which bins they are in, 
    in accordance with the bin's grid-index. 
    This improves the speed with which they can be processed.
```

Reply:
Review suggestions should go to https://github.com/pace-neutrons/Horace/pull/1689. This branch has been done to highlight changes and will not be merged